src/env.js

```diff
@@ -55,7 +55,7 @@ if (Array.isArray(skip)) {
 }
 else if (typeof skip === 'string') {
   const r = new RegExp(skip);
-  skip = s => s.test(r);
+  skip = s => s.match(r);
```

Comment:
Would still prefer test to avoid allocating the match object unnecessarily:

```suggestion
  skip = s => t.test(s);
```

Reply:
Done!